### PR TITLE
Darkmode Fixes

### PR DESCRIPTION
**Pull request by ovbm**
Small fixes for darkmode:
- Email Editorial links inherit color
- Interaction.A as alternative for linkRule // UNNECESSARY, A is already exported from Typography. needs to be removed.
- Scribble titles now darkmode ready

### CODE REVIEW

**Review by annatraussnig — APPROVED** [review]
🌕 🌔 🌑

File: src/templates/EditorialNewsletter/web/index.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ const createSchema = ({ ...args } = {}) => {
     List,
     ListItem,
     ListP,
+    Link: A,
```
Comment (ovbm) [inline src/templates/EditorialNewsletter/web/index.js:37]: Using this causes the app to crash: `Error: React.Children.only expected to receive a single React element child.`
Any Idea as to why? @tpreusse

File: src/components/Typography/Editorial.js [hunk 2]
```diff
@@ -265,16 +265,26 @@ export const link = css({
...
   const [colorScheme] = useColorContext()
-  const colors = css({
-    color: colorScheme.text,
-    '@media (hover)': {
-      ':hover': {
-        color: colorScheme.lightText
-      }
-    }
-  })
+  const AHoverRule = useMemo(
```
Comment (tpreusse) [inline src/components/Typography/Editorial.js:268]:
```suggestion
  const hoverRule = useMemo(
```

Variables starting with capital letters always smell of React components ;-)

**Review by tpreusse — APPROVED** [review]

**Pull request merged**